<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workspaceDlg</class>
 <widget class="QDialog" name="workspaceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Workspace</string>
  </property>
  <widget class="QLabel" name="newWorkspace">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New Workspace</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>471</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="size">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cell Size (Between 100 and 1000000):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>471</width>
     <height>85</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sizeText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>471</width>
     <height>28</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameText</tabstop>
  <tabstop>sizeText</tabstop>
  <tabstop>descriptionText</tabstop>
  <tabstop>cancel</tabstop>
  <tabstop>confirm</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>workspaceDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirm</sender>
   <signal>clicked()</signal>
   <receiver>workspaceDlg</receiver>
   <slot>finish()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>finish()</slot>
 </slots>
</ui>
